docs/learn-parachains.md

```diff
@@ -79,6 +80,71 @@ Reserved balances are non-transferrable and cannot be used for staking. At the e
+the Relay Chain for its primary function: validating parachains. Adding a system level chain could
+make the network capable of processing several more parachains. Rather than taking a slice of a 100
+parachain pie, a system level chain takes one slice and bakes a bigger pie.
+
+Examples of potential system level chains include parachains for balances, elections (for both
+staking and Council), governance, and identity. Eventually, the Relay Chain could become
+transactionless, as in, it would only validate parachain state transitions and all of its current
+transactional functionality would exist within parachains.
+
+The vast majority of common good chains will likely be the unopinionated system level chains.
+
+### Public Utility Chains
+
+Public utility chains add functionality that doesn’t exist yet, but that the stakeholders believe
+will add value to the entire network. Because public utility chains add new functionality, there is
+a subjective component to their addition: the stakeholders of the network must believe that it is
+worth allocating a slot that would otherwise go to the winners of an auction, and thus would have an
+objective expression of conviction from its backers. Governance provides the means to internalize
+the value of the parachain slot and distribute it across all members of the network.
+
+Public utility chains will always be fully aligned with their Relay Chain stakeholder base. This
+means that they will adopt the Relay Chain's native token (i.e. DOT or KSM) as their native token
+and respect any messages incoming from the Relay Chain and system level parachains at face value.
+
+Some examples of potential public utility chains are bridges, generic asset chains, and
+DOT-denominated smart contract platforms. All of these could operate without a new token:
+
+A bridge could add its own native token to charge as a toll, but in many cases that would be
+arbitrary value capture, when it could just as well use DOT and/or the bridged chain’s assets in its
+fee mechanism. A DOT-denominated smart contract layer-one blockchain would allow Wasm smart contract
+execution using DOT as the native asset with which to pay gas. A generic assets chain would allow
+anyone to place a deposit in DOT to deploy their asset on-chain. Assets on this chain could be
+backed by physical goods like artwork, real estate, or gold; or by paper goods like shares in a
+company or fiat currency held by a trusted party, providing a stable, permanent launchpad for
+stablecoins and Central Bank Digital Currencies. Public utility parachains would typically grant
+privileged business logic to Polkadot’s governance. Just as the Polkadot Relay Chain has several
+privileged functions like setting the validator count or allocating DOT from the Treasury, these
+parachains can have privileged functions like changing system parameters or registering an asset.
+
+Because public utility chains add functionality beyond the scope of the Relay Chain, we expect the
```

Comment:
We don't normally use first person in the Wiki, and here unfortunately it leads us into passive voice, like: "... public utility chains will likely be approved only in rare scenarios."

Reply:
I forgot 'we' is still first person!